app/modules/create-user/CreateUserScreen.tsx

```diff
@@ -32,21 +32,29 @@ const CreateUserScreen = () => {
     setModalShown,
     handleCameraSelect,
     handleGallerySelect,
-  } = useCreate({imagePath, setImagePath});
+  }: UseCreateReturnType = useCreate({imagePath, setImagePath});
+
+  const { theme } = useContext(ThemeContext);
+  const styles = stylesheet(theme as ThemeType);
```

Comment:
What is the need for typecasting 'as' here?

Reply:
because stylesheet parameter type is ThemeType and useContext saves theme from async storage sometimes that may be null or undefined(so ThemeType or undefined) and sometimes it has a theme that is changed within application via bottom sheet that has type ThemeType so to make stylesheet parameter compatible with it we have to typecast

Reply:
Make sure to do this at the initial or higher level, not at the lower level where at all places we are utilizing.